lib/tjbot.js

```diff
@@ -197,6 +197,7 @@ TJBot.prototype._setupCamera = function() {
         height: this.configuration.see.camera.height,
         noPreview: true,
         encoding: 'jpg',
+        outputDir: './',
```

Comment:
Above was changed so that takePhoto("test.jpg") saves the file to the current directory.
When no argument is provided, it is saved to /tmp.